webpack.config.common.js

```diff
@@ -214,12 +214,12 @@ module.exports = {
                 }
             },
             {
-                test: /\.woff(\?v=\d+\.\d+\.\d+)?$/,
+                test: /\.woff(2)?(\?v=\d+\.\d+\.\d+)?$/,
                 use: {
-                    loader: "url-loader?mimetype=application/font-woff",
+                    loader: "file-loader",
                     options: {
-                        name: '[name].[hash].[ext]',
-                        outputPath: 'font'
+                        name: '[name].[ext]',
```

Comment:
If you remove the `[hash] ` then the resource wouldn't be a versionable anymore, although the fonts are not changed often.